test/unit/Services/HttpServiceTest.php

```diff
@@ -169,6 +169,27 @@ static function ($headers) use ($locale) {
         $httpServiceMock->send('/my/uri/123', $resource);
     }
 
+    /**
+     * Verify 'CLIENTIP' header only set when a clientIp is defined in the Unzer object.
+     *
+     * @test
+     * @dataProvider clientIpHeaderShouldBeSetProperlyDP
+     *
+     * @param $clientIp
+     * @param mixed $isHeaderExpected
+     */
+    public function clientIpHeaderShouldBeSetProperly($clientIp, $isHeaderExpected): void
+    {
+        $unzer = new Unzer('s-priv-MyTestKey', $clientIp);
```

Comment:
The second parameter of the Unzer constructor is supposed to be the locale.
Please do not change that.